os-eap64-sso/module.yaml

```diff
@@ -7,9 +7,10 @@ execute:
   user: '185'
 
 artifacts:
-- path: rh-sso-7.2.2-eap6-adapter.zip
-  description: "The artifact is available on the Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=57851&product=core.service.rhsso&version=7.2&downloadType=securityPatches"
-  sha256: 739f2e92615bfd874c4b2c7fd8da2d5d4d8560bd853d459f9b9fcb131becb8f9
-- path: rh-sso-7.2.2-saml-eap6-adapter.zip
-  description: "The artifact is available on the Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=57811&product=core.service.rhsso&version=7.2&downloadType=securityPatches"
-  sha256: 27ace37d175ef3023cd5aa0dcdff8e54c86fc803fc8ef38d1ab7240cb006628a
+- name: rh-sso-7.3-eap6-adapter
+  target: rh-sso-7.3-eap6-adapter.zip
+  md5: dddbebeb28da106e7d4ff3e5a800fbe5
+- name: rh-sso-7.3-saml-eap6-adapter.zip
+- target: rh-sso-7.3-saml-eap6-adapter.zip
```

Comment:
@luck3y this dash is causing this error:
`Resource type is not supported: {'name': 'rh-sso-7.3-saml-eap6-adapter.zip'}`

Also, the name of the artifact can be just `rh-sso-7.3-saml-eap6-adapter`?